client/components/chart/utils.js

```diff
@@ -268,9 +268,6 @@ const getXTicksFromIncrementFactor = ( uniqueDates, incrementFactor ) => {
 	if ( ticks[ 0 ] !== uniqueDates[ 0 ] ) {
```

Comment:
We will have to update the comment right above this line to remove `or last`. :slightly_smiling_face: